frontend/const.go

```diff
@@ -17,10 +17,10 @@ const (
 	ContextKeyCorrelationData
 	ContextKeySystemData
 
-	// Wildcard path segment names for request multiplexing
+	// Wildcard path segment names for request multiplexing, must be lowercase as we lowercase the request URL before pattern matching
```

Comment:
Small nit with the wording, since this is for the URL pattern not the request URL itself.
```suggestion
	// Wildcard path segment names for request multiplexing, must be lowercase as we lowercase the request URL pattern when registering handlers
```